Python/Python.sublime-syntax

```diff
@@ -501,10 +501,11 @@ contexts:
     - include: allow-unpack-operators
 
   case-statement-end:
-    - match: ':(?!=)'
-      scope:
-        meta.statement.conditional.case.python
-        punctuation.section.block.conditional.case.python
+    # consume trailing whitespace so ST applies indentation rules reliably
```

Comment:
Does this still work with comments after the colon?

Reply:
Reindenting seems uneffected by comments after colons.

Indentation when hitting enter after `case "foo": # comment` completely fails. But this is unrelated to this change.

I won't work on that as all efforts to improve this kind of behaviors in other syntaxes has been completely reverted recently.

Reply:
The next commit should improve situation with trailing comments.